test/e2e/logforwarding/fluentnew/fluentnew_test.go

```diff
@@ -0,0 +1,129 @@
+package fluentnew
+
+import (
+	"fmt"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	"github.com/openshift/cluster-logging-operator/test/client/fluentd"
+	. "github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/object"
+	"github.com/openshift/cluster-logging-operator/test/object/clusterlogging"
```

Comment:
`object` is quite a generic pkg name. Why not calling it `crds`?

Reply:
Because corev1.Pod etc. are not CRDs, and because the central type in the API is called runtime.Object. I'm open to suggestions though, I had trouble deciding on this this. It's inconsistent as it stands because I gave clusterlogging a package of its own but no the other types.

My first take was:

```
package test
type Object struct{ o runtime.Object }
func (o Object) NamespacedName() ....
```

So you can say:

```
var o runtime.Object
obj := test.Object(o)
m := obj.Meta()
nn := obj.NamespacedName()
```

I was pretty pleased with myself, but in practice I almost always ended up only needing to call one method so it was usually:

```
nn := test.Object(o).NamespacedName()
```

in which case a simple package function is, well, simpler. I thought I might be getting fancy for no real gain (I do that sometimes) What do you think? I could be persuaded to go back to that, or if you have something simpler in mind...

Reply:
I suggest to improve identifying speed and lessen cognitive bias between apimachinery are code, why not replicate the package name as `runtime`. That buys us:
```
github.com/openshift/cluster-logging-operator/test/runtime/clusterlogging
```
which is easier to relate to
```
"k8s.io/apimachinery/pkg/runtime"
```
WDYT?

Reply:
+1 I like it

Reply:
Fixed